system_tests/lewis_emulators/Dg645/interfaces/stream_interface.py

```diff
@@ -2,9 +2,14 @@
 from lewis.core.logging import has_log
 from lewis.utils.command_builder import CmdBuilder
 
+from lewis_emulators.Dg645.device import SimulatedDg645
+
 
 @has_log
 class Dg645StreamInterface(StreamInterface):
+    def __init__(self) -> None:
+        self._device: SimulatedDg645
```

Comment:
i don't think this calls to the superclass. You might be able to type hint outside of `__init__()` to avoid this

Reply:
or if not we should call `super().__init__()`